### DISCUSSION TITLE
Create new GUI window

### DISCUSSION THREAD

**Discussion by dmocek** [neovide/neovide#2337] [General]
Is there a way to create a new GUI window using keyboard shortcuts?  I'd like to use Neovide like IntelliJ where I have a project in each GUI window and I'd like to be able to create a new GUI window, as if I just started Neovide, easily.

**Comment by fredizzimo** [comment]
At some point we will add support for multiple windows, in the way that is supported by Neovim. But I suspect that's not sufficient if you want to run multiple projects, since those windows will more be like a split showing a single buffer.

But you could create a keyboard mapping that launches `neovide` using [jobstrart()](https://neovim.io/doc/user/builtin.html#jobstart()). It would even be possible to open rpc channels for communication between the instances. And you could even take it even further with [:NeovideFocus](https://github.com/neovide/neovide/blob/9dde8013a64814b6962cb477897073fbad1ecbd2/website/docs/commands.md#focus-window)
